src/components/learning-guide-chart.coffee

```diff
@@ -3,44 +3,57 @@ d3 = require 'd3'
 
 CLOUDS_PATH = '/style/resources/clouds.svg'
 PLANE_PATH = '/style/resources/openstax-plane.svg'
+CITYSCAPE_PATH = '/style/resources/cityscape.svg'
+
+# should be svg bound to the label text, but temp for now.
+FLAG_BLUE = '/style/resources/flag-blue.png'
```

Comment:
Any reason these aren't drawn as SVG or included (with text) in the flight path background? Maybe this could be a separate PR?